torch/_inductor/padding.py

```diff
@@ -0,0 +1,63 @@
+import torch
+
+def needs_padding(args):
+  
+  for arg in args:
+    size = arg.meta["tensor_meta"].shape
+    for i in size:
+      if i // get_alignment_size(i) or i < 512:
```

Comment:
Looking at hte charts you developed 512 seemed to be where padding starts to matter